### PR TITLE
Adding form to mobile-version

### PR DESCRIPTION
**Pull request by yohannesdagnachew**
Adding a form to mobile version: 
1. Add a name field to accept up to 30 chars.
2. Make the email field accept only emails.
3. Message field accepts up to 500 chars.
4. Make all fields have placeholder text.
5. Make all fields have the correct type.
 6. A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -252,5 +252,66 @@ <h2 class="job">I’am a Software Developer</h2>
...
+        <li class="form-message">
+          <label for="message"></label>
+          <textarea
+            id="message"
+            name="user_message"
+            placeholder="Enter text here"
+            rows="8"
+            cols="37"
+            maxlength="500"
+          ></textarea>
```
Comment (aimemalaika) [inline index.html:291]: - [ ] Kindly make sure the textarea is required

File: index.html [hunk 2]
```diff
@@ -252,5 +252,66 @@ <h2 class="job">I’am a Software Developer</h2>
...
+            required
+          />
+          <img class="form-name-img" src="image/Container.svg" alt="" />
+        </li>
+        <li class="form-email">
+          <label class="user-email" for="email"></label>
+          <input
+            type="email"
+            id="email"
+            name="user_email"
```
Comment (aimemalaika) [inline index.html:276]: - [ ] Kindly center the form and work on the page responsivity on the mobile version
Screen Shot 2022-04-27 at 12 44 19

Comment (aimemalaika) [inline index.html:276]: - [ ] Kindly remove outlines of focus on each input as they are reducing the beauty of the design
Screen Shot 2022-04-27 at 12 46 46

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @yohannesdagnachew , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

 ## Required Changes ♻️

- [ ] I realized that there is some unrelated file in your project that contains SQL script, kindly remove the.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @yohannesdagnachew and @CrystallineButterfly,

Nice job making the required changes. Unfortunately, you missed some of them but you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please consider that this is a collaboration project and all collaborators suppose to push an equal amount of work but I can see that one of you did most of the work so kindly change that by doing the upcoming changes.

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Mre55 — APPROVED** [review]
**Hi @yohannesdagnachew,**

# Status: APPROVED  🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

Alt Text

**[OPTIONAL]** Please change your placeholder project name to your project name(i.e. Portfolio Project).

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**